alert-rules.md

```diff
@@ -2,6 +2,7 @@
 title: TiDB 集群报警规则
 summary: TiDB 集群中各组件的报警规则详解。
 category: reference
+aliases: ['/docs-cn/dev/reference/alert-rules.md']
```

Comment:
```suggestion
aliases: ['/docs-cn/dev/reference/alert-rules/']
```

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/3094/commits/22618ebc7d20e0654ebcf049bc19ce6743c9f791